index.html

```diff
@@ -4,44 +4,158 @@
-            don’t hesitate to contact me.</p>
-            <nav class="icon-wrap">
-                <ul class="navicon">
-                   <li><a href="https://github.com/Zuwairanajma"><img class="github" src="PImages/GitHub.png" alt="Github Icon"></a></li>
-                   <li><a href="https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?"><img class="linkedin" src ="PImages/Linkedin.png" alt = "Linkedin Icon"></a></li> 
-                   <li><a href="#"><img class ="angeli" src ="PImages/Vector3.png" alt="Angellist Icon"></a></li>
-                   <li><a href="https://twitter.com/@SadiqJuwairiyya"><img class ="twitter" src ="PImages/Twitter.png" alt ="Twitter Icon"></a></li>
-                   <li><a href="#"><img class ="medium" src="PImages/Vector5.png" alt ="Medium Icon"></a></li>
-                </ul>
-            </nav>
+            don’t hesitate to contact me.
+        </p>
+        <nav class="icon-wrap">
+            <ul class="navicon">
+                  <li><a href="https://github.com/Zuwairanajma"><img class= "github" src="PImages/GitHub.png" alt="Github Icon"></a></li>
+                  <li><a href="https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?"><img class= "linkedin"src="PImages/Linkedin.png" alt="Linkedin Icon"></a></li> 
+                  <li><a href="#"><img class= "angeli"src="PImages/VectorH.png" alt="Angellist Icon"></a> </li>
+                  <li><a href="https://twitter.com/@SadiqJuwairiyya"><img class= "twitter" src="PImages/Twitter.png" alt="Twitter Icon"></a></li>
+                  <li><a href="#"><img class= "medium" src="PImages/Medium.png" alt="Medium Icon"></a></li>
+            </ul>
+        </nav>
         </div>
     </section>
     <!-- Headline Section End -->
-    </main>
+  
+    <!-- Recent Works Section Start -->
+
+    <section id="My-Recent" class="My-Recent">
+        <div id="recent-works_container" class="recent-works">
+        <h2 id="my-work" class="my-work">My Recent Works</h2>
+      </div>
+      <div class="line-divider">
+        <hr>
+  </div>
+      <div>
+      <img class = "placeholder" src="PImages/Img Placeholder.png" alt=""></div>
+          <div id="multip" class="multipost">
+              <h3 id="multi" class="multi">Multi-Post Stories</h3>
+              <p id="stories" class="stories">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p></div>
+          <ul id="buttons1" class="buttonss1">
+             <li class="btn1 btn-primary">CSS</li>
+             <li class="btn1 btn-primary">HTML</li>
+             <li class="btn1 btn-primary btn-lg">BootStrap</li>
+             <li class="btn1 btn-primary">Ruby</li>
+          </ul>
+          <button id="seeproject" class="btn2 btn-primary btn-lg">See Project</button>
```

Comment:
- [ ] Kindly fix the size of this button to match the Figma design
- [ ] Remember that we need to follow the same design provided, you can use the properties on the Figma design and copy them in your CSS file. Kindly make your text color in the different `li` items be black as it on the Figma design

![image](https://github.com/Zuwairanajma/Portfolio-Projects/assets/93630700/0902db7f-1388-4a89-aded-b9ab6476f475)